client/js/transports/small-webrtc.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: "SmallWebRTCTransport"
+description: "A lightweight WebRTC transport for peer-to-peer connections with Pipecat"
+---
+
+The `SmallWebRTCTransport` class provides a WebRTC transport layer for connecting to Pipecat's `SmallWebRTCTransport` server-side implementation. It handles audio/video device management, WebRTC connections, and real-time communication between clients and bots.
+
+## Usage
+
+### Basic Setup
+
+```javascript
+import { RTVIClient } from "@pipecat-ai/client-js";
+import { SmallWebRTCTransport } from "@pipecat-ai/small-webrtc-transport";
+
+const transport = new SmallWebRTCTransport();
+
+const rtviClient = new RTVIClient({
+  transport,
+  enableCam: false, // Default camera off
+  enableMic: true, // Default microphone on
+  callbacks: {
+    // Event handlers
+  },
+  params: {
+    baseUrl: "http://localhost:7860",
+    endpoints: {
+      connect: "/api/offer",
+    },
+  },
+});
+
+await rtviClient.connect();
+```
+
+## API Reference
+
+### Constructor
+
+```typescript
+new SmallWebRTCTransport();
+```
+
+No parameters are required for basic functionality.
```

Comment:
the way this reads makes me think there are optional parameters you aren't telling me about.